src/components/dex/Chart.vue

```diff
@@ -457,6 +461,25 @@ export default {
         this.removeChart();
       }
     },
+
+    saveChartState(){
+      this.tradingView.onChartReady(() => {
+        this.tradingView.save((state) => {
```

Comment:
This saves all the settings for the chart, right? This shouldn’t need to write to storage if the data hasn’t changed though; since this is being called periodically, it should probably at least check if something changed before calling to save to storage.

Reply:
True, I didn't think it'd be an expensive operation saving to localStorage. But can add that check in prior to it doing it if you want?

Reply:
Might not matter that much, but let’s add it to reduce disk writes.

Reply:
Have done so in an additional [commit](https://github.com/aphtoken/aph-desktop-web-dex/commit/26fd5705ebb9f3661b6fae27f9afa2fc40555b7c).